micrometer-tracing-bridges/micrometer-tracing-bridge-brave/src/main/java/io/micrometer/tracing/brave/bridge/BraveBaggageManager.java

```diff
@@ -36,15 +37,25 @@ public class BraveBaggageManager implements Closeable, BaggageManager {
 
     private final List<String> tagFields;
 
+    private final List<String> remoteFields;
+
     @Nullable
     private Tracer tracer;
 
     /**
      * Create an instance of {@link BraveBaggageManager}.
      * @param tagFields fields of baggage keys that should become tags on a span
+     * @param remoteFields fields of baggage keys that should be propagated over the wire
      */
-    public BraveBaggageManager(List<String> tagFields) {
+    public BraveBaggageManager(List<String> tagFields, List<String> remoteFields) {
```

Comment:
Shouldn't we introduce a new ctor?
Can any of these be `null`?

Reply:
We can't - type inference is the same. They can't be null, we have non null api by default. We can pass empty collections